mavo-sort.js

```diff
@@ -0,0 +1,73 @@
+(function() {
+
+Mavo.Plugins.register("sort", {});
+
+Mavo.Functions.sort = function(array, ...properties) {
+	var arrayCopy = array.slice();
+	var splitChar = ".";
+
+	if (array.length === 0) {
+		return arrayCopy;
+	}
+
+	arrayCopy.sort(function(prev, next) {
+		// If it's an array of differnt types, we can't sort properly
+		if (typeof(prev) !== typeof(next)) {
+			return 0;
+		}
```

Comment:
Minor, but `typeof` is an operator in JS, not a function. You don't need parens.